mono/mini/aot-runtime.c

```diff
@@ -311,14 +311,14 @@ load_image (MonoAotModule *amodule, int index, MonoError *error)
 		assembly = mono_assembly_request_byname (&amodule->image_names [index], &req, &status);
 	}
 	if (!assembly) {
-		mono_trace (G_LOG_LEVEL_INFO, MONO_TRACE_AOT, "AOT: module %s is unusable because dependency %s is not found.", amodule->aot_name, amodule->image_names [index].name);
+		mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_AOT, "AOT: module %s is unusable because dependency %s is not found.", amodule->aot_name, amodule->image_names [index].name);
```

Comment:
Not sure about this one - it's  on an error path and seems like something unexpected.

Reply:
There are a few other spots on error paths.  Would it make sense to INFO in general on error?

Reply:
I think my emphasis was wrong - it's more that this seems like an unexpected situation (missing dependency) that warrants a higher log level, not just that it's an error path.